core/pkg/resourcehandler/k8sresources.go

```diff
@@ -147,6 +150,31 @@ func (k8sHandler *K8sResourceHandler) GetClusterAPIServerInfo() *version.Info {
 	return clusterAPIServerInfo
 }
 
+// set  namespaceToNumOfResources map in report
+func setMapNamespaceToNumOfResources(allResources map[string]workloadinterface.IMetadata, sessionObj *cautils.OPASessionObj) {
```

Comment:
You should set the context where we set the context and not here.

Reply:
can you please elaborate? This is the place where we have allResources and we insert info into cluster metadata, such as numberOfWorkers and etc